lib/helpers.js

```diff
@@ -61,3 +62,11 @@ function showError(givenMessage) {
     dismissable: true
   });
 }
+
+function validatePoint(textEditor, line, col) {
```

Comment:
Should this be `validateRange` and do similar logic but just use `TextBuffer::clipRange`? It would save calling it twice...